homeassistant/components/microsoft_face/__init__.py

```diff
@@ -98,7 +98,9 @@ async def async_create_group(service):
             entities[g_id] = MicrosoftFaceGroupEntity(hass, face, g_id, name)
             entities[g_id].async_write_ha_state()
         except HomeAssistantError as err:
-            _LOGGER.error("Can't create group '%s' with error: %s", g_id, err)
+            msg = f"Can't create group '{g_id}' with error: {err}"
```

Comment:
Please use old style formatting argument specifier for logging messages, as in the existing code.

Reply:
The logging message should not be broken out so we can avoid formatting it if necessary.